webpack.config.js

```diff
@@ -9,37 +9,42 @@ module.exports = {
         test: /\.js$/,
         exclude: /node_modules/,
         use: {
-          loader: "babel-loader"
-        }
+          loader: 'babel-loader',
+        },
       },
       {
         test: /\.html$/,
         use: [
           {
-            loader: "html-loader",
-            options: { minimize: true }
-          }
-        ]
+            loader: 'html-loader',
+            options: { minimize: true },
+          },
+        ],
       },
       {
         test: /\.css$/,
-        use: [MiniCssExtractPlugin.loader, "css-loader"]
-      }
-    ]
+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
+      },
+    ],
   },
   plugins: [
     new HtmlWebPackPlugin({
-      template: "./src/index.html",
-      filename: "./index.html"
+      template: './src/index.html',
+      filename: './index.html',
     }),
     new MiniCssExtractPlugin({
-      filename: "[name].css",
-      chunkFilename: "[id].css"
-    })
+      filename: '[name].css',
+      chunkFilename: '[id].css',
+    }),
   ],
   resolve: {
     alias: {
-      components: path.join(__dirname, 'src/js/components')
-    }
+      actions: path.join(__dirname, 'src/js/actions'),
```

Comment:
actually with actions.js file as well as constants.js file and reducers.js file, you should locate them on the same directory with container (for example: index.js file, style file, constants.js file, reducer.js file and a actions file of HomePage container should be located in the same directory with the name is HomePage, and HomePage should be located in containers directory)
Or if you prefer other directory structure, feel free to use it and discuss with me about its benefit, there are no standard structure for React Project but have some common structures, let's research about it, maybe there are some new ones which I haven't known yet :))